Formula/xapian.rb

```diff
@@ -101,6 +92,13 @@ def install
   end
 
   def caveats
+    if build.with? "python"
```

Comment:
It's been a while but I'm _fairly_ sure we have core logic to nag people to add Homebrew's site-packages directory to the system Python if necessary, which isn't the case for Ruby, hence the caveats there.